src/main/java/com/ecore/roles/exception/ErrorResponse.java

```diff
@@ -13,5 +13,6 @@ public class ErrorResponse {
 
     private final int status;
     private final String error;
+    private final ErrorDetail[] violations;
```

Comment:
Add Violations field to show bean validation errors in exception handler response representation